content/docs/state-and-lifecycle.md

```diff
@@ -68,25 +68,25 @@ ReactDOM.render(
 );
 ```
 
-To implement this, we need to add "state" to the `Clock` component.
+Para implementar esto, necesitamos agregar «estado» al componente `Clock`.
 
-State is similar to props, but it is private and fully controlled by the component.
+El estado es similar a las props, pero es privado y está completamente controlado por el componente.
```

Comment:
No biggie also, but, why are you treating `props` feminine? We always treat it as masculine because you tend to say `el objeto props` instead of `las propiedades`.

As say, no big of a deal and this should not prevent the merge of this PR - we can iterate on this later.

Reply:
I prefer "las props" because is short for "las propiedades" but wouldn't mind if we choose otherwise.